Support 'parameter type' in hierarchical blocks (#5309) (#5333)
verilator/verilator: docs/guide/exe_verilator.rst

   that the setting is ignored unless the :vlopt:`--hierarchical` option is
   specified.  See :ref:`Hierarchical Verilation`.

.. option:: hier_params -module "<modulename>"

>    Specifies that the module contains parameters for hier block. This option

Reviewer: ```suggestion
   Specifies that the module contains parameters a :vlopt:`--hierarchical` block. This option
```
